src/js/modules/articles/components/summaries/RibbonComponent.js

```diff
@@ -1,18 +1,18 @@
 import React from "react";
-import { connect } from "react-redux";
 import { Link } from "react-router-dom";
 
-const RibbonComponent = ({ classes, article, section, featuredMedia }) => {
+const RibbonComponent = ({ classes, article }) => {
```

Comment:
Why RibbonComponent? Why not Ribbon?

Reply:
There is already a RibbonRow that these are a part of. It's called the LatestArticlesRibbon.